chain/client/tests/process_blocks.rs

```diff
@@ -875,7 +873,6 @@ fn test_gc_block_skips() {
             env.produce_block(0, i);
         }
     }
-    assert!(check_refcount_map(&mut env.clients[0].chain).is_ok());
```

Comment:
why are these checks removed?

Reply:
They were added to sanity tests when there were not enough coverage of python tests and no Store Validator introduced. We don't need those checks anymore.

Reply:
I would still leave such asserts in the code, at least as `debug_assert!`. This can serve the documentation purpose and potentially help with refactorings if any will happen in the future.

Reply:
there is no 'check_refcount_map' anymore